backend/app/services/adapters/task_kinds/operations.py

```
@@ -949,6 +954,58 @@ def delete_all_archived_tasks(
+        user_id: int,
+        client_origin: Optional[str] = None,
+    ) -> int:
+        """Soft delete a list of tasks owned by a user."""
+        count = 0
+        for task_id in task_ids:
+            try:
+                self.delete_task(
+                    db=db, task_id=task_id, user_id=user_id, client_origin=client_origin
+                )
+                count += 1
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)